lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1299,7 +1300,8 @@ def setUp(self):  # pylint: disable=arguments-differ
         ('student_enrolled', 403),
         ('staff', 204),
         ('course_staff', 204),
-        ('community_ta', 204)
+        ('community_ta', 204),
+        ('admin', 204)
     )
     @ddt.unpack
     def test_access(self, user, status):
```

Comment:
Also can we verify if the team has been actually deleted? 
With something like 
```
self.assertEqual(team_list['count'], 1)
course_one_data = {'course_id': str(self.test_course_1), 'username': user}
team_list = self.get_teams_list(user=user, data=course_one_data)
self.assertEqual(team_list['count'], 0)
```

Reply:
You're right. The tests can be improved. I'd look into it and see if I can change them properly. 👍

Reply:
I actually rechecked the tests. It deletes an item and changes the status accordingly 
Line#1308 `self.delete_team(self.solar_team.team_id, status, user=user)`

Reply:
Let's discuss.